tests/console/yast2_apparmor.pm

```diff
@@ -104,11 +104,13 @@ sub run {
         wait_screen_change { send_key 'down' };
         save_screenshot;
         send_key(is_pre_15() ? 'alt-t' : 'alt-c');
+        wait_still_screen(stilltime => 5);
         if (is_pre_15()) {
-            wait_screen_change { send_key 'tab' };
-            wait_screen_change { send_key 'end' };    # we need to search for recent toggled element at the of the list
+            send_key 'tab';
+            wait_still_screen(stilltime => 5);
+            send_key 'end';
         }
-        wait_still_screen(stilltime => 2);
+        wait_still_screen(stilltime => 5);
```

Comment:
I wonder why there would be the need for a `wait_still_screen` at all. The subsequent `assert_screen` already waits for the screen to appear. Maybe the needle is not asserting that the screen is fully rendered yet?

Reply:
Not sure, will test that. 
This test was 'unstable' (fails random only on SLE12.3 and only in 'production') so it had to be timeouts, but ill be happy to test some tuning downs.